recipes/stereo-pipeline/recipe.yaml

```diff
@@ -0,0 +1,90 @@
+context:
+  name: stereo-pipeline
+  version: "3.5.1"
+
+package:
+  name: ${{ name|lower }}
+  version: ${{ version }}
+
+source:
+    - url: https://github.com/NeoGeographyToolkit/StereoPipeline/releases/download/${{ version }}/StereoPackage.tar.gz
+      sha256: fac0ec4ae371eafc30cbbe80c58c914842c08f30157c5d5763eab05404e56871
+
+build:
+  number: 0
+  skip: win
+  script: build.sh
+
+requirements:
+  build:
+    - ${{ compiler("c") }}
+    - ${{ compiler("cxx") }}
+    - ${{ compiler("fortran") }}
+    - ${{ stdlib("c") }}
+    - cmake >=3.15
+    - make
+    
+  host:
+    - ale
+    - csm
+    - ceres-solver
+    - eigen
+    - flann
+    - fftw
+    - geotiff
+    - gflags
+    - glog
+    - libboost-devel
+    - libgdal
+    - libblas * *openblas
+    - libopencv
+    - libjpeg-turbo
+    - libpng
+    - libpdal
+    - libtiff
+    - libxml2
+    - llvm-openmp
+    - mesalib
+    - parallel
+    - pcl
+    - protobuf
+    - qt-main
+    - qwt
+    - qhull
+    - rapidjson
+    - rocksdb
+    - tbb
+    - tbb-devel
+    - suitesparse
+    - superlu
+    - usgscsm
+    - visionworkbench
+    - vlfeat
+    - xerces-c
+
+  run:
+    - ale
+    - csm
+    - parallel
+    - usgscsm
+    - visionworkbench
+
+tests:
+  - script:
+    # Test loading a camera and doing image-to-ground and ground-to-image operations
+    - cam_test --image $PREFIX/share/tests/ctx.json --cam1 $PREFIX/share/tests/ctx.json --cam2 $PREFIX/share/tests/ctx.json
```

Comment:
That `tests/` subdirectory is a bit too general, might be worth renaming to something more package-specific, like `$PREFIX/share/stereo-pipeline/tests/*`.

Reply:
This is the test directory of the stereo-pipeline package that I am shipping. Libraries go to lib, binaries to bin, and tests go tests.